[2021-04-27 17:58:28,046] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.docker_op_tester 2021-04-25T04:00:00+00:00 [queued]>
[2021-04-27 17:58:28,086] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.docker_op_tester 2021-04-25T04:00:00+00:00 [queued]>
[2021-04-27 17:58:28,087] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-27 17:58:28,088] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-04-27 17:58:28,089] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-27 17:58:28,114] {taskinstance.py:1063} INFO - Executing <Task(DockerOperator): docker_op_tester> on 2021-04-25T04:00:00+00:00
[2021-04-27 17:58:28,121] {standard_task_runner.py:52} INFO - Started process 308 to run task
[2021-04-27 17:58:28,143] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'docker_sample', 'docker_op_tester', '2021-04-25T04:00:00+00:00', '--job-id', '97', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dockerOperator.py', '--cfg-path', '/tmp/tmphpjkukhm', '--error-file', '/tmp/tmpys24kpoo']
[2021-04-27 17:58:28,149] {standard_task_runner.py:77} INFO - Job 97: Subtask docker_op_tester
[2021-04-27 17:58:28,254] {logging_mixin.py:104} INFO - Running <TaskInstance: docker_sample.docker_op_tester 2021-04-25T04:00:00+00:00 [running]> on host 44e91a58ae1b
[2021-04-27 17:58:28,405] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=docker_sample
AIRFLOW_CTX_TASK_ID=docker_op_tester
AIRFLOW_CTX_EXECUTION_DATE=2021-04-25T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-25T04:00:00+00:00
[2021-04-27 17:58:28,439] {docker.py:224} INFO - Starting docker container from image centos:latest
[2021-04-27 17:58:59,276] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=docker_sample, task_id=docker_op_tester, execution_date=20210425T040000, start_date=20210427T175828, end_date=20210427T175859
[2021-04-27 17:58:59,313] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-04-27 17:58:59,348] {local_task_job.py:146} INFO - Task exited with return code 0
